docs/operate/get-started/other-hardware/manage-modules.md

```diff
@@ -416,6 +416,42 @@ To delete just one version of a module:
 Versions are immutable, meaning that you cannot upload a new file with the same version number as the deleted one.
 To upload another version, you must increment the version number to a later version number.
 
+## Transfer ownership of a module
+
+To transfer ownership of a module from one organization to another:
+
+1. You must be an [owner](/manage/manage/rbac/) in both the current and new organizations.
+
+1. Navigate to the module's page in the [**REGISTRY** section of the Viam app](https://app.viam.com/registry).
+
+1. Make sure the visibility of the module is set to **Public**.
+
+1. Click the **...** menu in the upper-right corner of the page, and click **Transfer ownership**.
+
+1. Select the new organization from the dropdown menu, then click **Transfer module**.
+
+1. (Recommended) Transfer the GitHub repository containing the module code to the new owner.
+   Be sure to remove the existing secrets from the repository’s settings before transferring.
+   If the repository is using Viam’s cloud build, the secrets contain an organization API key that will be exposed to the new owner after the repository transfer.
+
+1. Update the `meta.json` file to reflect the new organization:
+
+   - Change the first part of the `module_id` field to the new organization's [namespace](/operate/reference/naming-modules/#create-a-namespace-for-your-organization).
+   - For each model, change the first part of the `model` field to the new organization's namespace.
+   - Update the `url` field to point to the new code repository if it has moved.
+
+1. Update the module code:
+
+   - Throughout your module implementation code, change the model names in your component or service classes to match the changes you made to the `meta.json` file.
+
+1. Run `viam module update` to push the changes to the registry.
+
+1. Publish a new version of the module to the registry by following either set of update steps on this page.
+
+## Rename a module
+
+If you need to change the name of a module, please reach out to the Viam team at [support@viam.com](mailto:support@viam.com).
+
 ## Next steps
```

Comment:
Out of diff but please remove the next steps. Since we have the next button at the bottom, this feels out of place. It also seems odd after info about transfering a module. This link could be with updating an existing module.

Reply:
Okay! We also have this link up top so yeah removing